enlighten/Measurement.py

```diff
@@ -329,6 +331,8 @@ def __init__(self,
         else:
             raise Exception("Measurement requires exactly one of (spec, source_pathname, measurement, dict)")
 
+        self.prefix = self.save_options.prefix() if self.save_options is not None else ""
+        self.suffix = self.save_options.suffix() if self.save_options is not None else ""
```

Comment:
As with the other PR, I'd probably move this under "if spec:" to avoid stomping over loaded files, and also default these to None in clear().

Reply:
Is there a reason for None as opposed to "". I'm thinking "" so it will still be a string and shouldn't throw an error when writing. I think None may throw an error.